modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/streaming/PagingProviderProducer.java

```diff
@@ -69,15 +71,17 @@ public final class PagingProviderProducer<T> implements Producer<List<T>> {
   public PagingProviderProducer(PagingProvider<Object, T> delegate,
                                 ConfigurationInstance config,
                                 ExecutionContextAdapter executionContext,
-                                ExtensionConnectionSupplier extensionConnectionSupplier) {
-    this(delegate, config, executionContext, extensionConnectionSupplier, false);
+                                ExtensionConnectionSupplier extensionConnectionSupplier,
+                                InitialSpanInfo initialSpanInfo) {
```

Comment:
There is a discrepancy between the name of the new parameter in this constructor and the same parameter in the other constructor of the same class. `initialSpanInfo` vs `getConnectionInitialSpanInfo`.
Not sure which one is the best one, but they certainly look like they should be called the same.